rivelinojunior/e_social_mine_api
# SETUP: generate rails project

1. Generate a new Rails API project with specific configurations using the provided command:

```bash
rails new PROJECT_NAME --api  \
                      -d=postgresql \
                      --skip-jbuilder \
                      --skip-action-text \
                      --skip-asset-pipeline \
                      --skip-action-cable \
                      --skip-action-mailbox \
                      --skip-test
```

2. Navigate to the folder created
`cd PROJECT_NAME`

3. Create a new branch:
`git checkout -b setup/generate_project`

4. Add the files generated at git stage
`git add .`

6. Commit the files:
`git commit -m 'SETUP: generate rails project'`

7. Push change to github:
`git push origin setup/generate_project`

8. Open the PR
[Follow this tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

> Before doing this task, make sure that you added the remote origing:

 ` git remote add origin git@github.com:rivelinojunior/e_social_mine_api.git`
